packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -44,8 +57,44 @@ export class ConfigUtil {
         const rootPath = getRootWorkspacePath();
         const myLocalConfig = await ConfigFile.create({
           isGlobal: false,
-          rootFolder: path.join(rootPath, '.sf'),
-          filename: 'config.json'
+          rootFolder: path.join(rootPath, SFDX_STATE_FOLDER),
```

Comment:
getConfigValue is only kept here for use by LibraryBaseTemplateCommand.  Once Templates is upgraded, this can be removed.

Reply:
should we make a ticket so we don't lose track of this?

Reply:
[W-11528745](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE0000131BVsYAM/view)